src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -581,27 +615,33 @@ blockToMarkdown' opts t@(Table _ blkCapt specs thead tbody tfoot) = do
   return $ nst (tbl $$ caption'') $$ blankline
 blockToMarkdown' opts (BulletList items) = do
   contents <- inList $ mapM (bulletListItemToMarkdown opts) items
-  return $ (if isTightList items then vcat else vsep) contents <> blankline
+  variant <- asks envVariant
+  return $ (if isTightList items || variant == Markua then vcat else vsep) 
+                  contents <> blankline
```

Comment:
Can you explain this?  Is it that list items *can't* have blank lines between them in Markua?

Reply:
We changed it accordingly